react/features/conference/components/web/Conference.js

```diff
@@ -117,7 +117,8 @@ class Conference extends AbstractConference<Props, *> {
         super(props);
 
         this.state = {
-            showDeeplink: true
+            showDeeplink: true,
+            timeout: true
```

Comment:
Looks like the changes in this file are no longer needed, please revert so it don't conflict with #222